lib/modules/dosomething/dosomething_campaign/dosomething_campaign.module

```diff
@@ -757,11 +757,17 @@ function dosomething_campaign_add_signup_data_form_vars(&$node, $langcode) {
 
   // Load the signup entity.
   $signup = signup_load($sid);
+
   // Pass to the user signup data form.
   $node->content['signup_data_form'] = drupal_get_form('dosomething_signup_user_signup_data_form', $signup);
+
   // If the signup_data_form is required and user has not submitted form yet:
-  $hasCompetitionRecordedLocally = $signup->signup_data_form_timestamp;
-  $shouldShowCompetitionPrompt = $config['required'] && !$hasCompetitionRecordedLocally;
+  if (!variable_get('rogue_collection', FALSE)) {
```

Comment:
I'd be up for just always doing the "if Rogue is enabled" thing now, since we're gonna enable it so soon.